proto/directions_next.proto

```diff
@@ -1,100 +0,0 @@
-syntax = "proto2";
```

Comment:
i think its safe to say we will never use this. @dgearhart do you disagree?

Reply:
agreed